en/style/colors.md

```diff
@@ -26,7 +28,21 @@ On the canvas below you will see a collection of colored circles representing al
 
 The same procedure can be followed for the `secondary` colors and other colors with special meaning in the library.
 
-## Creating Custom Colors
+## Palette Generation in Adobe XD
+
+To change the primary palette in Adobe XD open the Indigo.Design library file and show the `Libraries` panel. Then, right-click on `primary.500` and choose `Edit` from the contextual menu that appears to specify a different color. You will notice that this will update the palette on the `🎨 Colors` artboard but only the `primary.500` color in the `Libraries` panel will reflect the change.
```

Comment:
```suggestion
To change the primary palette in Adobe XD open the Indigo.Design library file and show the `Libraries` panel. Then, right-click on `primary.500` and choose `Edit` from the contextual menu that appears to specify a different color. You will notice that this will update the palette on the `🎨 Colors` artboard, but only the `primary.500` color in the `Libraries` panel will reflect the change.
```